com.unity.netcode.gameobjects/Documentation~/components/core/networkobject.md

```diff
@@ -58,6 +61,9 @@ To give ownership back to the server use the `RemoveOwnership` method:
 ```csharp
 GetComponent<NetworkObject>().RemoveOwnership();
 ```
+> [!NOTE]
+> Using `RemoveOwnership` in a distributed authority network topology isn't recommended.
```

Comment:
We actually throw an error in `RemoveOwnership` with DA ([NetworkSpawnManager.cs:L427](https://github.com/Unity-Technologies/com.unity.netcode.gameobjects/blob/d8f96d65ab90471893226766401b46059dd054f6/com.unity.netcode.gameobjects/Runtime/Spawning/NetworkSpawnManager.cs#L427))

```suggestion
> `RemoveOwnership` isn't supported when using a [distributed authority network topology](../../terms-concepts/distributed-authority.md).
```